Source/WebKit/UIProcess/Cocoa/WebProcessPoolCocoa.mm

```diff
@@ -922,9 +922,9 @@ static void logProcessPoolState(const WebProcessPool& pool)
 #endif
 
     m_finishedMobileAssetFontDownloadObserver = [[NSNotificationCenter defaultCenter] addObserverForName:@"FontActivateNotification" object:nil queue:[NSOperationQueue currentQueue] usingBlock:^(NSNotification *notification) {
-        NSString *fontFamily = notification.userInfo[@"FontActivateNotificationFontFamilyKey"];
+        RetainPtr<NSString> fontFamily = notification.userInfo[@"FontActivateNotificationFontFamilyKey"];
```

Comment:
It’s interesting that call sites like this are a way to write unchecked type casts. I wonder what our coding style and verification strategy should be. With C++ types we don’t want any unchecked type casts because of the security implications, but maybe it’s not as critical for Objective-C types.

Reply:
@rniwa It seems our static analysis doesn't currently warn for implicit unchecked casts in ObjC.

Reply:
The weirdest thing is that the next line is actually checking the type, *after* the implicit cast :) I'll update to use a dynamic_objc_cast here.